### PR TITLE
07 post /api/reviews/review id/comments

### PR DESCRIPTION
**Pull request by CJOROURKE96**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -149,4 +149,32 @@ describe('App', () => {
         });
     })
 })
+describe('POST /api/reviews/:review_id/comments', () => {
+    it('should return something', () => {
```
Comment (HeatherEver) [inline __tests__/app.test.js:153]: change this test name please 😂

File: __tests__/app.test.js [hunk 2]
```diff
@@ -149,4 +149,32 @@ describe('App', () => {
...
+        const input = {
+            username: "bainesface",
+            body: "This game is great"
+        }
+        return request(app)
+        .post('/api/reviews/2/comments')
+        .send(input)
+        .expect(201)
+        .then(({body}) => {
+            console.log(body.comment, "<-- BODY ")
```
Comment (HeatherEver) [inline __tests__/app.test.js:163]: remove console logs

File: __tests__/app.test.js [hunk 3]
```diff
@@ -149,4 +149,32 @@ describe('App', () => {
...
+        return request(app)
+        .post('/api/reviews/1000/comments')
+        .expect(404)
+    });
+    it('should return a 400 when passed anything that is not a number', () => {
+        return request(app)
+        .post('/api/reviews/abcd/comments')
+        .expect(400)
+    });
+});
```
Comment (HeatherEver) [inline __tests__/app.test.js:179]: Nice tests so far! At some point it would be nice to add tests for if the comment object is empty/has the wrong keys/invalid username

**Review by HeatherEver — APPROVED** [review]
Lovely work Connor, would be good to add a couple more error handling tests at some point :)
